test/client_test.go

```diff
@@ -134,7 +134,7 @@ func TestClientUniqueNames(t *testing.T) {
 
 func TestOpenedClientConnectionCount_WhenMultipleMembers(t *testing.T) {
 	cluster, _ = remoteController.CreateCluster("", DefaultServerConfig)
-	for i := 0; i < 5; i++ {
+	for i := 0; i < 2; i++ {
```

Comment:
why decrease?

Reply:
opening 5 members and closing them that a long time. It was taking 40 seconds. That is why I decreased it.